.github/actions/analyze-test-runs/action.yml

```diff
@@ -81,10 +81,10 @@ runs:
         if [ -s "$tmpFile" ]; then  # found tests
           sort "$tmpFile" | uniq -d | tee "$outputFile"
           if [ -s "$outputFile" ]; then
-            echo "::error::Found duplicate test runs!"
+            echo "::warning::Found duplicate test runs!"
             echo "### ⚠️ Duplicate test runs" >> $GITHUB_STEP_SUMMARY
             cat $outputFile >> $GITHUB_STEP_SUMMARY
-            exit 1
+            exit 0
```

Comment:
Hm, the check was there because we have had problems where we had multiple engines and the tests ended up running multiple times :sweat_smile: I suppose we can remove it as incidence is low, but if we keep it, it should be fail if: there are duplicate tests, and there are no flaky tests.

Not sure how easy it is to get that here :thinking:

Reply:
That said we haven't had problems with duplicate tests in a long time, so perhaps we can just remove it :shrug: